test/src/youbora.spec.js

```diff
@@ -41,7 +41,7 @@ describe('YouboraAdapter', function () {
     analyticsParams.player.should.equal(playerName);
     analyticsParams.username.should.equal(user);
     (analyticsParams.referer === document.referrer || analyticsParams.referer === location.href).should.be.true;
-    analyticsParams.pluginVersion.should.equal('6.7.35' + '-' + __VERSION__ + '-' + __NAME__);
+    analyticsParams.pluginVersion.should.equal('6.7.42' + '-' + __VERSION__ + '-' + __NAME__);
```

Comment:
There must be a better way to get this data from the plugin, also we should not really care about the fact this is the SDK version so maybe just do a regex to check that our `'-' + __VERSION__ + '-' + __NAME__` are part of the request